remote/functions.php

```diff
@@ -159,6 +159,14 @@ function remote_preload_webfonts() {
 
 add_action( 'wp_head', 'remote_preload_webfonts' );
 
-// Add block patterns
-//require get_template_directory() . '/inc/block-patterns.php';
+/**
+ * Block Patterns.
+ */
+require get_template_directory() . '/inc/block-patterns.php';
+
+// Add the child theme patterns if they exist.
+if ( file_exists( get_stylesheet_directory() . '/inc/block-patterns.php' ) ) {
+	require_once get_stylesheet_directory() . '/inc/block-patterns.php';
+}
```

Comment:
Remote is not a child theme, I think the require on line 165 is enough

Reply:
Removed!